app/modules/entities/components/layouts/entity_claim_link.svelte

```diff
@@ -1,20 +1,20 @@
 <script>
-  import { externalIdsUrlFormatters } from '#entities/lib/entity_links'
+  import { externalIdsDisplayConfigs } from '#entities/lib/entity_links'
   import Link from '#lib/components/link.svelte'
 
-  export let label, property, value
+  export let name, property, value
 
-  const formatter = externalIdsUrlFormatters[property]
+  const { getUrl } = externalIdsDisplayConfigs[property]
   let url
-  if (formatter) url = externalIdsUrlFormatters[property](value)
+  if (getUrl) url = getUrl(value)
 </script>
 
 {#if url}
   <!-- TODO: set an icon instead of a text -->
   <Link
     {url}
-    text={label}
-    title={label}
+    text={name}
```

Comment:
Im slightly confused, i thought `name` would ideally be reserved for human or organisation, while `label` is the [WD naming for an entity](https://www.wikidata.org/wiki/Wikidata:Glossary#Label). So, in the context of entity claim link, shouldn't this variable called `label` instead of `name` ?

Reply:
I used services `name`s to distinguish it from their specific properties labels. For instance, `wdt:P1085`'s English label is "LibraryThing work ID", `wdt:P7400`'s English label is "LibraryThing author ID", but both will be displayed with the name "LibraryThing" on their respective layouts (work and author layouts), as the full property label is more verbose than needed.